python/job_runner/reporting/file.py

```diff
@@ -0,0 +1,184 @@
+import json
+import os
+import shutil
+import socket
+import time
+
+from job_runner.io import cond_unlink
+from job_runner.reporting.message import Exited, Finish, Init, Running, Start
+
+
+class File(object):
+    LOG_file = "JOB_LOG"
+    ERROR_file = "ERROR"
+    EXIT_file = "EXIT"
+    STATUS_file = "STATUS"
+    OK_file = "OK"
+    STATUS_FILE = "status.json"
+
+    def __init__(self):
+        self.status_dict = {}
+        self.node = socket.gethostname()
+
+    def report(self, msg, sync_disc_timeout=10):
```

Comment:
File reporter has different signature(... , msg, .. ) Interactive and network ( ... , status, ... )